packages/commonwealth/client/scripts/views/pages/CommunityGroupsAndMembers/Groups/Update/UpdateCommunityGroupPage.tsx

```diff
@@ -39,6 +35,22 @@ const UpdateCommunityGroupPage = ({ groupId }: { groupId: string }) => {
     (group) => group.id === parseInt(`${groupId}`),
   );
 
+  const initialAllowlist = useMemo(() => {
+    return foundGroup?.requirements
+      .filter((r) => r?.rule === 'allow') // Filter only the allowlist rules
```

Comment:
what are the possible values for `.rule`, if its an enum, then let's use a enum type for strings ex: `allow`

Reply:
I think we can't enum this, because rule can depend on the chain type which is dynamic. It just so happens it also applies to allowlist rules as well.